Src/RE2.Managed/RE2Regex.cs

```diff
@@ -15,7 +14,9 @@ public class RE2Regex : IRegex
         public static IRegex Instance = new RE2Regex();
         public static TimeSpan DefaultTimeout = TimeSpan.FromMilliseconds(int.MaxValue - 1);
 
-        private RE2Regex() { }
+        private RE2Regex()
+        {
+        }
```

Comment:
Is this needed?

Reply:
my format automatically makes this change. will revert later.